ale/drivers/dawn_drivers.py

```diff
@@ -105,7 +106,7 @@ def target_name(self):
         : str
           target name
         """
-        target = self.label['TARGET_NAME']
+        target = super().target_name
```

Comment:
@jlaura @jessemapel Does this not seem like a better tool to use than managing private and public versions of certain methods to avoid the infinite recursion issue...? And if so, do we still need to implement private methods anywhere in ALE?